source/blog/2011-01-19-has-the-future-turned-out-the-way-it-was-supposed-to.html.markdown

```diff
@@ -9,7 +9,7 @@ tags:
   - Rails
 ---
 
-<p><img alt="Does Not Compute?!" src="http://i1291.photobucket.com/albums/b548/grammccram/display_size_Has-the-Future.._zpsqjz3pohb.jpg" style="width: 358px; height: 563px;" /><img alt="" src="file:///Users/tomdickinson/Library/Caches/TemporaryItems/moz-screenshot.png" /></p>
```

Comment:
While we're tidying things up, we can remove the second img tag here with it's `file://` url.